style.css

```diff
@@ -122,3 +128,222 @@ header {
 .social {
   height: 18px;
 }
+
+/* Works section */
+
+.works {
+  padding-bottom: 30px;
+  display: grid;
+  background-color: #e5e5e5;
+}
+
+.works-grid-item {
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  border-radius: 16px;
+  padding: 16px;
+  min-height: 100px;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  margin-bottom: 50px;
+  margin-left: 10px;
+  margin-right: 14px;
+}
```

Comment:
- [ ] Kindly make sure to place this projects cards in the center in order to meet the provided design as requested by the previous reviewer. 

  - ### Your design 
 
![design](https://user-images.githubusercontent.com/87197412/185664397-21b93ba8-e4bf-46bf-9c05-3f823117f48d.PNG)

  - ### Figma design
   
![image](https://user-images.githubusercontent.com/87197412/185664458-be54f3f1-f1d9-4a1a-916c-f75c5b8314ee.png)